code/src/App.js

```diff
@@ -1,9 +1,38 @@
-import React from 'react'
+import React from "react";
+import { Provider } from "react-redux";
+import { combineReducers, createStore } from "@reduxjs/toolkit";
+
+import tasks from "./reducers/tasks";
+import Wrapper from "components/Wrapper";
+
+const reducer = combineReducers({
+  tasks: tasks.reducer,
+});
+
+// -- add local storage
+
+const persistedStateJSON = localStorage.getItem("reduxState");
+let persistedState = {};
+
+if (persistedStateJSON) {
+  persistedState = JSON.parse(persistedStateJSON);
+}
+
+const store = createStore(
+  reducer,
+  persistedState,
+  // -- to make the redux devtools work
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
Did they go through this somewhere because I totally have missed that this was supposed to be in the code =D